diff --git a/xpad.c b/xpad.c
--- a/xpad.c
+++ b/xpad.c
@@ -446,6 +446,7 @@ static const struct usb_device_id xpad_table[] = {
        { USB_DEVICE(0x0738, 0x4540) },        /* Mad Catz Beat Pad */
        XPAD_XBOXONE_VENDOR(0x0738),           /* Mad Catz FightStick TE 2 */
        XPAD_XBOX360_VENDOR(0x07ff),           /* Mad Catz GamePad */
+       XPAD_XBOX360_VENDOR(0x0db0),           /* Micro Star International controllers */
        XPAD_XBOX360_VENDOR(0x0c12),           /* Zeroplus X-Box 360 controllers */
        XPAD_XBOX360_VENDOR(0x0e6f),           /* 0x0e6f X-Box 360 controllers */
        XPAD_XBOXONE_VENDOR(0x0e6f),           /* 0x0e6f X-Box One controllers */
